tests/contexts/index.js

```diff
@@ -1,16 +1,20 @@
 const credentialsContext = require('credentials-context');
+const didContext = require('did-context');
+
 const invalidContexts = {
   veresOne: {
     url: 'https://w3id.org/veres-one/v1',
     value: require('./veresOnev1.json')
   },
   did: {
     url: 'https://w3id.org/did/v0.11',
-    value: require('./didv0.json')
+    value: didContext.contexts.get('https://w3id.org/did/v0.11')
```

Comment:
rather than using the string literal for the URL (which is subject to change) use the const exported and defined here : https://github.com/digitalbazaar/did-context/blob/master/js/constants.js#L3